migrate.py

```diff
@@ -56,6 +64,26 @@
 )
 move.add_argument("new_path", help="The destination of the file/folder")
 
+# Add the "changeLink" subcommand
+change_link = subparsers.add_parser(
+    "changeLink",
+    help="Open the database and swap the system paths only without migrating files",
+)
+change_link.add_argument(
+    "old_path",
+    help="The old system path to be changed",
+)
+change_link.add_argument(
+    "new_path",
+    help="The new system path to replace the old one",
+)
+change_link.add_argument(
+    "--replace-slashes",
+    action="store_true",
+    default=False,
+    help="Replace backslashes with forward slashes in the database paths",
+)
+
```

Comment:
If you're support Windows -> Linux, can you also add Linux -> Windows (_probably_ less likely, but seems worthwhile to add)